solutions/swb-reference/integration-tests/support/resources/datasets/datasets.ts

```diff
@@ -28,17 +28,17 @@ export default class Datasets extends CollectionResource {
     return {
       datasetName: resource.datasetName ?? dataSetName,
       path: resource.path ?? dataSetName,
-      storageName: resource.storageName,
-      awsAccountId: resource.awsAccountId,
-      region: resource.region
+      description: resource.description,
+      customMetadata: resource.customMetadata
     };
   }
 }
 
 interface DataSetCreateRequest {
   datasetName: string;
-  storageName: string;
   path: string;
-  awsAccountId: string;
-  region: string;
+  description?: string;
+  customMetadata: {
```

Comment:
why `customMetadata.owningProjectId` and not just `owner` (similar to what the actual API expects)?

Reply:
It makes more sense for UI to call it owningProjectId, but for more generic dataset service we would keep it as owner